Public/Sdk/Public/Transformers/Transformer.ReadPipGraphFragment.dsc

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Transformer {
+
+    /**
+     * Read in a fragment of the graph as a binary file and all all pip from it to the pip graph.
+     * name: Name of this pip graph fragment, used to return a handle to it.

```

Comment:
>name [](start = 7, length = 4)

Why do we need to specify a name? It's a bit deviating from DScript where we typically only relies on returned value.

Reply:
is there a way to get the variable name in the c#?  I could use that instead.

---
In reply to: [289573083](https://github.com/microsoft/BuildXL/pull/394#discussion_r289573083) [](ancestors = 289573083)